src/pokemon.c

```diff
@@ -2087,6 +2087,17 @@ u32 GetMonData2(struct Pokemon *mon, s32 field)
     return GetMonData3(mon, field, NULL);
 }
 
+struct EvolutionTrackerBitfield {
+  u16 a: 5;
+  u16 b: 4;
+  u16 unused: 7;
+};
+
+union EvolutionTracker {
+  u16 value;
+  struct EvolutionTrackerBitfield asField;
+};
```

Comment:
4-spaces, braces in new line.
```suggestion
struct EvolutionTrackerBitfield
{
    u16 a: 5;
    u16 b: 4;
    u16 unused: 7;
};

union EvolutionTracker
{
    u16 value;
    struct EvolutionTrackerBitfield asField;
};
```